src/main/features/claim/routes/amount.ts

```diff
@@ -22,18 +22,28 @@ function addErrorMessage (form: Form<Amount>, fieldName: string, errorMessage: s
   form.errors.push(new FormValidationError(validationError, ''))
 }
 
+function handleWhenAnyValueAndCannotStateSelected (form: Form<Amount>): void {
+  const higherValue: number = form.model.higherValue
+  const lowerValue: number = form.model.lowerValue
+  const higherValueEntered = higherValue != null && higherValue > 0
+  const lowerValueEntered = lowerValue != null && lowerValue > 0
+  const canNotStateSelected = form.model.cannotState === Amount.CANNOT_STATE_VALUE
+  const onlyLowerValueEntered = (lowerValueEntered && !higherValueEntered)
+  const CanNotStateAndAnyValueEntered = (higherValueEntered || onlyLowerValueEntered) && canNotStateSelected
```

Comment:
please rename `CanNotStateAndAnyValueEntered` to `canNotStateAndAnyValueEntered`